docs/content/install.sh

```diff
@@ -9,9 +9,23 @@ if [ -n "$1" ] && [ "$1" != "beta" ]; then
 fi
 
 if [ -n "$1" ]; then
-    install_beta="yes"
+    install_beta="beta "
 fi
 
+#check installed version of rclone to determine if update is necessary
+version=`rclone --version | head -n 1`
```

Comment:
I haven't checked, but I think this will emit worrying error messages if rclone doesn't exist...

I think you probably want `rclone --version 2>/dev/null | head -n 1` but I haven't tested that either :-)

Reply:
@ncw 
Your solution work. 
When rclone is not installed the version variable will be empty.

Reply:
Thanks for pointing this out. Unfortunately, I think that null device is not pre-installed on BSD systems and maybe the user doesn't want it, so installing it in this script wouldn't be a good idea.
To fix this problem I'll just move to the created temporary directory earlier and redirect the errors to a new file in it.